src/v2/BaseLoginRouter.js

```diff
@@ -116,7 +116,7 @@ export default Router.extend({
     // transform response
     const ionResponse = transformIdxResponse(this.settings, idxResponse, lastResponse);
 
-    this.appState.setIonResponse(ionResponse);
+    this.appState.setIonResponse(ionResponse, this.controller);
```

Comment:
Didn't feel good if we have to pass around previous `controller`. 
Maybe we miss something somewhere?

Reply:
needs `controller` context to trigger errors from AppState, right before we change the app state.

Reply:
removed this.